src/main/java/net/tropicraft/core/common/entity/underdasea/atlantoku/EntityTropicraftWaterBase.java

```diff
@@ -579,6 +595,21 @@ protected void resetHeight() {
 	public int getTalkInterval() {
 		return 120;
 	}
+	
+	@Override
+	protected SoundEvent getAmbientSound() {
+		return SoundEvents.ENTITY_SQUID_AMBIENT;
```

Comment:
We should find dolphin sounds eventually

Reply:
pinch your lips and pull air in